src/nls/root/shortcuts-win.js

```diff
@@ -38,6 +38,7 @@ define(function (require, exports, module) {
                 SAVE: "s",
                 SAVE_AS: "s",
                 REVERT: "KEY_F12",
+                EXPORT_UTILITY: "e",
```

Comment:
Are these keyboard shortcuts cleared with @placegraphichere? I just want to make sure this doesn't conflict with some important PS shortcut...